ansible/roles/bigbluebutton/defaults/main.yml

```diff
@@ -0,0 +1,19 @@
+---
+
+galaxy_path: "{{ playbook_dir }}/roles.galaxy"
+# Git commit or release version of bbb_role_git
+bbb_role_ver: 5f1e22498921e10f2c18bfb4a311aede18a2b62f
+# https://github.com/n0emis/ansible-role-bigbluebutton
+bbb_galaxy_role: n0emis.bigbluebutton
+
+# https://github.com/geerlingguy/ansible-role-nodejs/releases
+nodejs_galaxy_role: geerlingguy.nodejs
+nodejs_role_ver: 5.1.1
+# User for whom the npm packages will be installed, defaults to ansible_user.
+nodejs_install_npm_user: npmuser
+# https://github.com/geerlingguy/ansible-role-docker/releases
+docker_galaxy_role: geerlingguy.docker
+docker_role_ver: 2.8.2
+# https://github.com/geerlingguy/ansible-role-pip/releases
+pip_galaxy_role: geerlingguy.pip
+pip_role_ver: 2.0.0
```

Comment:
I know you like to keep this idempotent and all, but are we going to remeber to update these dependencies? I doubt it. So I would prefer if we didn't specify the version of these, even though idempotence goes to hell.. Either way, we should assume these are well tested enough....

Reply:
This has not much to do with idempotency the versions are always required and we should take care to be up to date.
I updated one upstream role to the latest version in 71e085f.

Reply:
I think it does. My assumption here is that there is a way to set the version to 
`latest` which will on one hand mean we don't have to remember to update these every time we deploy. On the other hand we loose "more" idempotency since when we run it again some of those roles might have been updated and be broken and thus break everything else.

Do you understand what I mean?

Reply:
By using the latest version, we have no idea of what version we are running on our server.
This means that we are going to get a different version when we deploy the role in specific times.

Resulting in very hard debugging, if we ever get into a situation to resolve an issue related to the upstream roles.

Reply:
Exactly. (that's essentially one way to loose idempotence). But my point being: will we/you remember to update these dependencies? or will they be left forgotten and deprecated.

My point being, making it "latest" although may make harder debugging it will end up being safer. But your call.

Reply:
We should "remember" to update these roles once we update the upstream's role version.
Not knowing what version of a role we are running will bite us back on a bug in the near future. ;)